Tests/UITests/Tests.cs

```diff
@@ -68,54 +70,95 @@ public void TestEnablingAndDisablingServices()
         }
 
         [Test]
-        public void TestServiceStatePersistence()
+        public void TestServiceStatePersistenceCrashes()
         {
+            app.Screenshot("App Launched - Ready for tests");
             ServiceStateHelper.app = app;
+            app.WaitForElement(TestStrings.GoToTogglePageButton);
             app.Tap(TestStrings.GoToTogglePageButton);
 
             /* Make sure Crashes enabled state is persistent */
             ServiceStateHelper.AppCenterEnabled = true;
             ServiceStateHelper.CrashesEnabled = false;
             Assert.IsFalse(ServiceStateHelper.CrashesEnabled);
+            //There seems to be some sort of timing issue here.
+            //   without the Thread.Sleep - this code fails although I see that the method called is Async (with a Wait)
+            //   so I think this deserves further investigation. I've added a terribly long wait to get the tests green.
+            Thread.Sleep(10000);
             app = AppInitializer.StartAppNoClear(platform);
+            app.WaitForElement(TestStrings.GoToTogglePageButton);
             app.Tap(TestStrings.GoToTogglePageButton);
             Assert.IsTrue(ServiceStateHelper.AppCenterEnabled);
             Assert.IsTrue(ServiceStateHelper.AnalyticsEnabled);
             Assert.IsFalse(ServiceStateHelper.CrashesEnabled);
+            app.Screenshot("Crashes persistent");
+
+            /* Reset services to enabled */
+            ServiceStateHelper.AppCenterEnabled = true;
+        }
+
+        [Test]
+        public void TestServiceStatePersistenceAnalytics()
+        {
+            app.Screenshot("App Launched - Ready for tests");
+            ServiceStateHelper.app = app;
+            app.WaitForElement(TestStrings.GoToTogglePageButton);
+            app.Tap(TestStrings.GoToTogglePageButton);
 
             /* Make sure Analytics enabled state is persistent */
             ServiceStateHelper.AppCenterEnabled = true;
             ServiceStateHelper.AnalyticsEnabled = false;
             Assert.IsFalse(ServiceStateHelper.AnalyticsEnabled);
+            Thread.Sleep(10000);
```

Comment:
Please move these sleep values to constants in this file.  Align if possible or name appropriately.